packages/rum-recorder/src/domain/rrweb-snapshot/snapshot.ts

```diff
@@ -426,13 +426,10 @@ export function serializeNodeWithId(
   return serializedNodeWithId
 }
 
-export function snapshot(n: Document): [SerializedNodeWithId | null, IdNodeMap] {
-  const idNodeMap: IdNodeMap = {}
-  return [
-    serializeNodeWithId(n, {
-      doc: n,
-      map: idNodeMap,
-    }),
-    idNodeMap,
-  ]
+export function serializeDocument(n: Document): SerializedNodeWithId {
```

Comment:
should we rename this file then or is snapshot still relevant?
or do you plan to address that with later refactorings?

Reply:
Yes I plan to address it later, because the whole `rrweb-snapshot` is not really relevant anymore either.